salt/_utils/vault.py

```diff
@@ -0,0 +1,116 @@
+# -*- coding: utf-8 -*-
+
+from __future__ import absolute_import, print_function, unicode_literals
+import base64
+import logging
+import os
+import requests
+import json
+import time
+import yaml
+import hvac
```

Comment:
This doesn't cause import errors if hvac is not available?

Reply:
No, it doesn't. `import hvac` is checked in the `vault.module` already, and only when successfully imported, `vault.utils` is called at later calls.